<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smackdown Vs Raw Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="wrestlerLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="wrestlerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Wrestler</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="wrestlerGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>771</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Wrestler Info</string>
    </property>
    <widget class="QGroupBox" name="imageGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>115</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Picture</string>
     </property>
     <widget class="QLabel" name="wrestlerImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: 3px solid black;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="attributesGroupBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>321</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Attributes</string>
     </property>
     <widget class="QLabel" name="grappleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Grapple</string>
      </property>
     </widget>
     <widget class="QLabel" name="strikeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Strike</string>
      </property>
     </widget>
     <widget class="QLabel" name="submissionLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Submission</string>
      </property>
     </widget>
     <widget class="QLabel" name="durabilityLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Durability</string>
      </property>
     </widget>
     <widget class="Line" name="attributesLine">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>16</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="techniqueLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Technique</string>
      </property>
     </widget>
     <widget class="QLabel" name="speedLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="charismaLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Charisma</string>
      </property>
     </widget>
     <widget class="QLabel" name="hardcoreLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Hardcore</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="grappleLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="strikeLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="submissionLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="durabilityLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="techniqueLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="speedLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="charismaLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="hardcoreLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Requirements</string>
    </property>
    <widget class="QCheckBox" name="chCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CH.pac</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch2CheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CH2.pac</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chetcCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CHETC.pac</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch3CheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CH3.pac</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="bgCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>BG.pac</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="menuCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MENU.pac</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="miscCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MISC.pac</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="plistPs2ArcCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PLISTPS2.arc</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="plistPs2hCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PLISTPS2.h</string>
     </property>
    </widget>
    <widget class="QLabel" name="directoryLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Files Directory :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="direcotryLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
